### PR TITLE
Docs - isIPv4String, isIPv6String functions

### PR DESCRIPTION
**Pull request by bharatnc**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

Detailed description / Documentation draft:

This PR documents and explains the use of the functions `isIPv4String`
and `isIPv6String`.

Related to: #18349

### CODE REVIEW

File: docs/en/sql-reference/functions/ip-address-functions.md [hunk 1]
```diff
@@ -243,4 +243,37 @@ SELECT
...
 
+
+## isIPv4String
+
+Determines if the input string is an IPv4 address or not. Returns `1` if true `0` otherwise.
+
+``` sql
+SELECT isIPv4String('127.0.0.1')
+
+Query id: 031fdb77-fa98-456e-9323-c68c7788fdde
```
Comment (vdimir) [inline docs/en/sql-reference/functions/ip-address-functions.md:254]: Lets' remove `Query id` and `1 rows in set. Elapsed:` from examples. Also we can split query and response in separate code block`sql` and `text` as in examples above.

Comment (bharatnc) [reply docs/en/sql-reference/functions/ip-address-functions.md]: Sure, fixed..

**Comment by vdimir** [comment]
Thank you for your PR

**Comment by bharatnc** [comment]
> Thank you for your PR

@vdimir you're welcome!

**Pull request merged**